tilepix.go

```diff
@@ -105,11 +114,18 @@ func Read(r io.Reader, dir string) (*Map, error) {
 	log.WithField("Tileset count", len(m.Tilesets)).Debug("Read: checking for tileset sources")
 	for i, ts := range m.Tilesets {
 		if ts.Source != "" {
-			sourceTs, err := readTilesetFile(filepath.Join(dir, ts.Source))
+			f, err := openFileFunc(filepath.Join(dir, ts.Source))
+			if err != nil {
+				log.WithError(err).Error("Read: could not read tileset source")
+				return nil, err
+			}
+			sourceTs, err := readTileset(f, dir)
```

Comment:
We always close after this call anyway, only need to call `.Close()` twice.
Also, prefer to explicitly ignore returned errors with a blank assignment for visibility
```suggestion
			sourceTs, err := readTileset(f, dir)
			_ = f.Close()
```